### PR TITLE
Create a new schema versions for the change of spaceless ids

### LINKED ISSUE #212: Update schema versions and docs for spaceless ID change
make the change in new v0.0.3  instead of 0.0.2

- [ ]  copy everything from 0.0.2 into 0.0.3 folder, and then change 0.0.2 in each file to 0.0.3, update schemas.md (the README) and then change the default to be 0.0.3
- [ ] update schemas.md  with the info on the change and when it was made for 0.0.9 and 0.1.0

### PR DESCRIPTION
**Pull request by jgreben**
Fixes #212 

- Create a new schema versions 0.0.3 and 0.1.1 for the change of spaceless ids
- Rolls back v0.1.0 to original schema state
- documents change in schemas

### CODE REVIEW

File: schemas/0.0.3/profile.json [hunk 1]
```diff
@@ -0,0 +1,79 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://ld4p.github.io/sinopia/schemas/0.0.2/profile.json",
```
Comment (ndushay) [inline schemas/0.0.3/profile.json]: oops

File: schemas/0.0.3/profile.json [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+            "http://sinopia.io/resources/common/AdminMetadataProfile"
+          ]
+        },
+        "remark": {
+          "type": "string",
+          "title": "Remark",
+          "description": "Comment or guiding statement intended to be presented as supplementary information in user display."
+        },
+        "resourceTemplates": {
+          "$ref": "https://ld4p.github.io/sinopia/schemas/0.0.2/resource-templates-array.json"
```
Comment (ndushay) [inline schemas/0.0.3/profile.json]: oops

File: schemas/0.0.3/profiles-array.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "https://ld4p.github.io/sinopia/schemas/0.0.2/profiles-array.json",
```
Comment (ndushay) [inline schemas/0.0.3/profiles-array.json]: oops

File: schemas/0.0.3/profiles-array.json [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+  "title": "LC Profile Array Schema (i.e. Multiple Profiles) (by Sinopia)",
+  "version": "0.0.3",
+  "type": "array",
+  "items": {
+    "$ref": "https://ld4p.github.io/sinopia/schemas/0.0.2/profile.json"
```
Comment (ndushay) [inline schemas/0.0.3/profiles-array.json]: oops

File: schemas/0.0.3/property-template.json [hunk 5]
```diff
@@ -0,0 +1,177 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://ld4p.github.io/sinopia/schemas/0.0.2/property-template.json",
```
Comment (ndushay) [inline schemas/0.0.3/property-template.json]: oops

File: schemas/0.0.3/property-templates-array.json [hunk 6]
```diff
@@ -0,0 +1,11 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "https://ld4p.github.io/sinopia/schemas/0.0.2/property-templates-array.json",
```
Comment (ndushay) [inline schemas/0.0.3/property-templates-array.json]: oops

File: schemas/0.0.3/property-templates-array.json [hunk 7]
```diff
@@ -0,0 +1,11 @@
...
+  "title": "LC Property Template Array Schema (i.e. Multiple Property Templates) (by Sinopia)",
+  "version": "0.0.3",
+  "type": "array",
+  "items": {
+    "$ref": "https://ld4p.github.io/sinopia/schemas/0.0.2/property-template.json"
```
Comment (ndushay) [inline schemas/0.0.3/property-templates-array.json]: oops

File: schemas/0.0.3/resource-template.json [hunk 8]
```diff
@@ -0,0 +1,62 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://ld4p.github.io/sinopia/schemas/0.0.2/resource-template.json",
```
Comment (ndushay) [inline schemas/0.0.3/resource-template.json]: oops

File: schemas/0.0.3/resource-template.json [hunk 9]
```diff
@@ -0,0 +1,62 @@
...
+      "title": "Resource Template Identifier",
+      "description": "Identifier associated with a resource template. Eventually, a URI.",
+      "pattern": "^\\S+$",
+      "example": [
+        "profile:bf2:AdminMetadata",
+        "http://sinopia.io/resources/common/AbstractResourceTemplate"
+      ]
+    },
+    "propertyTemplates": {
+      "$ref": "https://ld4p.github.io/sinopia/schemas/0.0.2/property-templates-array.json"
```
Comment (ndushay) [inline schemas/0.0.3/resource-template.json]: oops

File: schemas/0.0.3/resource-templates-array.json [hunk 10]
```diff
@@ -0,0 +1,11 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "https://ld4p.github.io/sinopia/schemas/0.0.2/resource-templates-array.json",
```
Comment (ndushay) [inline schemas/0.0.3/resource-templates-array.json]: oops

File: schemas/0.0.3/resource-templates-array.json [hunk 11]
```diff
@@ -0,0 +1,11 @@
...
+  "title": "LC Resource Template Array Schema (i.e. Multiple Resource Templates) (by Sinopia)",
+  "version": "0.0.3",
+  "type": "array",
+  "items": {
+    "$ref": "https://ld4p.github.io/sinopia/schemas/0.0.2/resource-template.json"
```
Comment (ndushay) [inline schemas/0.0.3/resource-templates-array.json]: oops

File: schemas/0.1.1/profile.json [hunk 12]
```diff
@@ -0,0 +1,119 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://ld4p.github.io/sinopia/schemas/0.1.0/profile.json",
```
Comment (ndushay) [inline schemas/0.1.1/profile.json]: etc.

**Comment by jgreben** [comment]
@ndushay thanks for looking over my shoulder! I think I fixed all the refs at this point.

> I'm thinking maybe v0.1.1 could just be v0.2.0 since it's not backwards compatible. what do you think?

That sounds fine to me. I will push a commit.

> And of course, once we've got this PR merged and deployed, you'll want to update the editor to recognize 0.2.0, 0.1.0, 0.0.9 and default to 0.0.3?

I know what you mean about changing the default to 0.0.3, but I'm not 100% sure what you mean by "recognizing" all the other schemas. Wouldn't it just be a matter of changing all the existing references from 0.1.0 to 0.2.0?

**Comment by ndushay** [comment]
>> And of course, once we've got this PR merged and deployed, you'll want to update the editor to recognize 0.2.0, 0.1.0, 0.0.9 and default to 0.0.3?

>I know what you mean about changing the default to 0.0.3, but I'm not 100% sure what you mean by "recognizing" all the other schemas. Wouldn't it just be a matter of changing all the existing references from 0.1.0 to 0.2.0?

I can't remember the editor code well enough;  just thinking that we want the editor to be able to cope with 0.2.0, 0.1.0, 0.0.9 and 0.0.3  profiles/resource templates.   0.2.0 and 0.1.0 and 0.0.9 resource templates will have the schema url inside them;  0.0.3 templates do not.   (Was there a pre-loading of schemas into the ajv object?)  -- this would be a separate ticket b/c it's in the editor codebase.

File: schemas.md [hunk 13]
```diff
@@ -22,6 +22,10 @@ Property Template:
 - move dataTypeURI out two levels, instead of valueConstraints.valueDataType.dataTypeURI or maybe include it within default, if it is only used for (type lookup with a ?) default.
 - only accept boolean (not string) for true/false.  (e.g. true not "true")
 
+## Version 0.1.1
```
Comment (ndushay) [inline schemas.md]: oops - 0.2.0 here too.

**Pull request merged**